### PR TITLE
updated utilStages to util_stages

### PR DESCRIPTION
**Pull request by eacharles**
## Change Description

- [ ] My PR includes a link to the issue that I am addressing

## Code Quality
- [ ] I have read the Contribution Guide
- [ ] My code follows the code style of this project
- [ ] My code builds (or compiles) cleanly without any errors or warnings
- [ ] My code contains relevant comments and necessary documentation

### Bug Fix Checklist
- [ ] My fix includes a new test that breaks as a result of the bug (if possible)
- [ ] My change includes a breaking change
  - [ ] My change includes backwards compatibility and deprecation warnings (if possible)

### New Feature Checklist
- [ ] I have added or updated the docstrings associated with my feature using the [NumPy docstring format](https://numpydoc.readthedocs.io/en/latest/format.html)
- [ ] I have updated the tutorial to highlight my new feature (if appropriate)
- [ ] I have added unit/End-to-End (E2E) test cases to cover my new feature
- [ ] My change includes a breaking change
  - [ ] My change includes backwards compatibility and deprecation warnings (if possible)

### Documentation Change Checklist
- [ ] Any updated docstrings use the [NumPy docstring format](https://numpydoc.readthedocs.io/en/latest/format.html)

### Build/CI Change Checklist
- [ ] If required or optional dependencies have changed (including version numbers), I have updated the README to reflect this
- [ ] If this is a new CI setup, I have added the associated badge to the README

### Other Change Checklist
- [ ] Any new or updated docstrings use the [NumPy docstring format](https://numpydoc.readthedocs.io/en/latest/format.html).
- [ ] I have updated the tutorial to highlight my new feature (if appropriate)
- [ ] I have added unit/End-to-End (E2E) test cases to cover any changes
- [ ] My change includes a breaking change
  - [ ] My change includes backwards compatibility and deprecation warnings (if possible)

### CODE REVIEW

**Comment by eacharles** [comment]
Fixes outstanding errors in unit tests coming from change of utilStages to util_stages and from downloading sfd dustmap.

File: tests/astro_tools/test_core.py [hunk 1]
```diff
@@ -22,11 +22,11 @@
 from rail.core.stage import RailStage
 from rail.tools.util_photometry import HyperbolicMagnitudes, HyperbolicSmoothing, PhotormetryManipulator, LSSTFluxToMagConverter, Dereddener
 from rail.core.utils import RAILDIR
-#from rail.tools.utilStages import (
+#from rail.tools.util_stages import (
```
Comment (OliviaLynn) [inline tests/astro_tools/test_core.py:25]:
```suggestion
#from rail.tools.util_photometry import (
```

**Review by sschmidt23 — APPROVED** [review]

File: tests/astro_tools/test_core.py [hunk 2]
```diff
@@ -52,21 +52,25 @@ def test_dereddener():
     testFile = os.path.join(RAILDIR, "rail", "examples_data", "testdata", "rubin_dm_dc2_example.pq")
     test_data = DS.read_file("test_data", TableHandle, testFile)
 
+    fluxToMag = LSSTFluxToMagConverter.make_stage(name='flux2mag', copy_cols=dict(ra='ra', decl='decl'))
+
     dustmap_dir = os.environ.get('RAIL_DUSTMAP_DIR')
-    is_temp_dir = False
```
Comment (OliviaLynn) [inline tests/astro_tools/test_core.py:56]: If we take this out, we might get to the `if is_temp_dir:` line without having defined `is_temp_dir`, no? (or do we get this for free with pytest?)

**Comment by sschmidt23** [comment]
whoops, hit return before typing, looks good, thanks Eric.

File: tests/astro_tools/test_core.py (hunk 1 above)
Comment (eacharles) [reply tests/astro_tools/test_core.py:25]: Thanks.  Nice catch.   FWIW, these were already merged into main with #https://github.com/LSSTDESC/rail_astro_tools/pull/23/files.  it looks like I didn't rebase to main for this PR.

File: tests/astro_tools/test_core.py (hunk 2 above)
Comment (eacharles) [reply tests/astro_tools/test_core.py:56]: yes, you are right.  we should fix this.

**Pull request merged**